### PR TITLE
Upgrade to net6

### LINKED ISSUE #101: Upgrade to net6.0

### PR DESCRIPTION
**Pull request by sprucely**
Closes #101

### CODE REVIEW

**Comment by jSylvestre** [comment]
Still get the failed to load resources using node 18
main.js and vendor.js

File: azure-pipelines.yml [hunk 1]
```diff
@@ -4,21 +4,21 @@ trigger:
 name: 1.1.$(Date:yyMM).$(build.buildId)
 
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: 'windows-latest'
```
Comment (srkirkland) [inline azure-pipelines.yml:7]: I'd prefer ubuntu if at all possible.  it runs more than 2x as fast usually

**Review by srkirkland — APPROVED** [review]
haven't tried but code looks good

File: src/CaesHelp/Startup.cs [hunk 2]
```diff
@@ -12,11 +13,11 @@
 using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.HttpsPolicy;
 using Microsoft.AspNetCore.Mvc;
-using Microsoft.AspNetCore.SpaServices;
+using Microsoft.AspNetCore.SpaServices.ReactDevelopmentServer;
```
Comment (jSylvestre) [inline src/CaesHelp/Startup.cs:16]: Development?

**Pull request merged**